.github/workflows/release.yml

```diff
@@ -0,0 +1,67 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Generate Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASEBOT_TOKEN }}
+          GIT_AUTHOR_NAME: "service-github-lyft-semantic-release"
+          GIT_AUTHOR_EMAIL: "service-github-lyft-semantic-release@users.noreply.github.com"
+          GIT_COMMITTER_NAME: "service-github-lyft-semantic-release"
+          GIT_COMMITTER_EMAIL: "service-github-lyft-semantic-release@users.noreply.github.com"
+        run: npx semantic-release
```

Comment:
I was hoping we can use an already available github action that does this.... but I guess that's ok

Reply:
We could always publish this as a separate GH action to make it more reusable?

Reply:
yeah I think that would be cool.. this also updates the code right? not just the github release? if you ever get to make it a github action I would also make it output the new version so that we can simplify the code below...